src/index.js

```diff
@@ -147,6 +201,7 @@ class Libp2p extends EventEmitter {
     })
 
     if (this._config.relay.enabled) {
+      // @ts-ignore Circuit prototype
       this.transportManager.add(Circuit.prototype[Symbol.toStringTag], Circuit)
```

Comment:
We use `Circuit.prototype[Symbol.toStringTag]` as a way to identify transports, so that we provide in libp2p config transport options as:

```js
transports: {
  [Circuit.prototype[Symbol.toStringTag]]: { // .... }
}
```

Not sure if this is what you were looking for?